notebooks/02_vector_zonal_stats.ipynb

```diff
@@ -19,7 +19,10 @@
    "source": [
     "# hide\n",
     "# no_test\n",
-    "! [ -e /content ] && pip install -Uqq git+https://github.com/thinkingmachines/geowrangler.git"
+    "! [ -e /content ] && pip install -Uqq git+https://github.com/thinkingmachines/geowrangler.git\n",
```

Comment:
The general idea is to make it easy to explore the whole package, not just the tutorials, but even the implementation details. For users who might be evaluating the package if its suitable for their needs, having the ability to test this out quickly can mean the difference between adoption instead of building their own. 

Also, my own experience with nbdev is that having the ability to run everything in colab is also useful in reproducing and fixing bugs, especially if I can click on it and modify the behavior to see my idea works.  As an example, see how [fastai](https://docs.fast.ai/) and [fastcore](https://fastcore.fast.ai/) document sites leverage this capability.

This ability to explore and try ideas out is an advantage, because it also lowers the barriers for contributing. If they find a fix or a better implementation quickly and be able to test it out in colab, it will encourage them to do the whole extra effort of setting up all the rest (cloning the repo, setting up a jupyter environment on their local, etc). See the nbdev section on [developing on colab](https://nbdev.fast.ai/tutorial_colab.html) which focuses on this.

Reply:
when running fastapi in collab it even throws the following error
```
---------------------------------------------------------------------------

ModuleNotFoundError                       Traceback (most recent call last)

[<ipython-input-3-7106bed3169c>](https://localhost:8080/#) in <module>()
      1 #|hide
----> 2 from nbdev.showdoc import *

ModuleNotFoundError: No module named 'nbdev'


---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------
```

Reply:
Rather than working on each notebook to work instantly on colab, maybe we can just include docs on common fixes if they face issues when running things in collab.